### PR TITLE
[python/all] Changing Python to PythonLegacy, and Python3 to Python

### PR DESCRIPTION
**Pull request by PetitPrince**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

I think there is no reason for a newcomer to learn Python2 instead of the current version of Python. Furthermore:
 * Python 2 will not be maintained past 2020 ([countdown clock](https://pythonclock.org/))
 * [359 out of the 360 most downloaded package in PyPi are compatible Python 3](http://py3readiness.org/)
 * [In the majority of case, Python 3 is faster than Python  2](https://hackernoon.com/which-is-the-fastest-version-of-python-2ae7c61a6b2b)

This pull request implement [Aaron Maxwell (powerfulpython) idea](https://powerfulpython.com/blog/magic-word-legacy-python/) of calling the second version of Python "legacy Python" instead of "Python 2" (and all other version of Python just "Python").

This will make newcomer naturally go to the non-legacy version first, and into the legacy if need be.

### CODE REVIEW

**Comment by divayprakash** [comment]
@adambard Could you please weigh in on this? Going through the links above, I think it's a worthwhile change.

File: python.html.markdown [hunk 1]
```diff
@@ -1,100 +1,95 @@
...
-Note: This article applies to Python 2.7 specifically, but should be applicable
-to Python 2.x. Python 2.7 is reaching end of life and will stop being
-maintained in 2020, it is though recommended to start learning Python with
-Python 3. For Python 3.x, take a look at the [Python 3 tutorial](http://learnxinyminutes.com/docs/python3/).
-
-It is also possible to write Python code which is compatible with Python 2.7
-and 3.x at the same time, using Python [`__future__` imports](https://docs.python.org/2/library/__future__.html). `__future__` imports
-allow you to write Python 3 code that will run on Python 2, so check out the
-Python 3 tutorial.
+Note: This article applies to Python 3 specifically. Check out [here](http://learnxinyminutes.com/docs/python/) if you want to learn the old Python 2.7
```
Comment (adambard) [inline python.html.markdown]: This link needs updating

**Comment by adambard** [comment]
I approve this change in principle, but the PR needs some extra work -- mostly, the many translations of python and python3 articles being updated similarly so that all our links don't break.

So, I'm going to leave this open either until I have a moment to do that, or until it is otherwise addressed

**Comment by adambard** [comment]
Upon further consideration I don't love the idea of swapping out the existing python article. I think URLs should remain consistent over time (as does Google, incidentally).

Could I convince you as a compromise to move the `python` file to `legacypython` but leave the python 3 one as is? That way I can just nginx-redirect /python to /legacypython and solve that issue. It'll also make the translation thing I said before much less work.

**Comment by PetitPrince** [comment]
It would stills itch me a bit that https://learnxinyminutes.com/docs/python/ would redirect to the Python2 doc (why wouldn't this redirect to the latest version ?), but I understand the translation issues and have updated the PR accordingly.

I plan to update all the translations and so I updated the name of the PR accordingly. Should I create a PR for each language or is OK as it is ?

**Comment by adambard** [comment]
Any sites linking to /python would be expecting the python 2 doc, so I think it's appropriate to leave that redirect in place, at least until 2's deprecation is official.

By all means feel free to update the translations in this PR, I don't want to deal with dozens of branches any more than you :)

**Comment by adambard** [comment]
@PetitPrince thanks for doing all that! I noticed that python3 es-es was missing, but everything else is there. Are you ok with merging this in as-is? (I can just go in and edit the missing file after merging)

**Comment by thecraftman** [comment]
@adambard @divayprakash @severin @PetitPrince @vitaliel  is it okay also to learn python 2.7?

**Comment by thecraftman** [comment]
wow interesting .

**Comment by adambard** [comment]
@thecraftman I would absolutely recommend python 3 wherever possible at this point. Python 2 won't vanish overnight, but Python 3-compatible libraries etc. are going to be the ones receiving the most maintenance.

**Comment by thecraftman** [comment]
@adambard  thanks . i have started learning python 3 tho it has been awesome. looking forward to learn from you too

**Comment by heitorPB** [comment]
[Python 2 is no more maintained](https://www.python.org/doc/sunset-python-2/)

IMO, python should now refer to python3.

still interested in changing this?

**Comment by divayprakash** [comment]
> [Python 2 is no more maintained](https://www.python.org/doc/sunset-python-2/)
> 
> IMO, python should now refer to python3.
> 
> still interested in changing this?

@adambard Thoughts?

**Comment by adambard** [comment]
@divayprakash I agree, we should now point "Python" at python 3.

**Comment by justjosias** [comment]
Now that this PR is long outdated and the changes have been made in #3856. should this be closed?

**Pull request closed without merging**